File: RepoTransBench/target_projects/Java/C#/mohak1712_UberUX/public_tests/getPolylinePublicTest.cs
using Xunit;
using System.Drawing;

namespace UberUX.Tests.Public
{
    public class getPolylinePublicTest
    {
        [Fact]
        public void TestDecodePolyline_DifferentCoords_Public()
        {
            // Test input
            string testPolyline = "_p~iF~ps|U_ulLnnqC_mqNvxq`@";
            var result = UberUX.getPolyline.decodePolyStatic(testPolyline);

            Assert.True(result.Count >= 2);
        }
    }
}

File: RepoTransBench/target_projects/Java/Go/mohak1712_UberUX/public_tests/get_polyline_public_test.go
package public_tests

import (
	"testing"
	"mohak1712_uberux/tests"
	"github.com/stretchr/testify/assert"
)

func TestDecodePolylineDifferentCoordsPublic(t *testing.T) {
	g := &tests.GetPolyline{}
	testPolyline := "_p~iF~ps|U_ulLnnqC_mqNvxq`@"
	result := g.DecodePoly(testPolyline)
	assert.True(t, len(result) >= 2)
}